src/cloud-api-adaptor/test/e2e/common.go

```diff
@@ -26,6 +27,14 @@ const BUSYBOX_IMAGE = "quay.io/prometheus/busybox:latest"
 const WAIT_DEPLOYMENT_AVAILABLE_TIMEOUT = time.Second * 180
 const DEFAULT_AUTH_SECRET = "auth-json-secret-default"
 
+func isTestWithKbs() bool {
+	return os.Getenv("DEPLOY_KBS") == "true" || os.Getenv("DEPLOY_KBS") == "yes"
```

Comment:
This is much better, thank you. Do you think we should https://github.com/confidential-containers/cloud-api-adaptor/blob/16768ab6ba2606b6d504d2f7e10c70cbccf553b6/src/cloud-api-adaptor/test/tools/provisioner-cli/main.go#L39-L42 to use this as well? It would probably mean moving this method into the test/provisioner package so that both can reference it?

Reply:
It's in different package/main program, maybe we should refactor provisioner-cli separately?

Reply:
Yeah, we can spin that out